config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -1,24 +1,47 @@
 presets:
 - labels:
-    preset-e2e-gce-windows: "true"
+    preset-common-gce-windows: "true"
   env:
   - name: KUBE_GCE_ENABLE_IP_ALIASES
     value: "true"
-  - name: NUM_WINDOWS_NODES
-    value: "3"
-  - name: NUM_NODES
-    value: "2"
   - name: KUBERNETES_NODE_PLATFORM
     value: "windows"
   - name: KUBELET_TEST_ARGS
     value: "--feature-gates=KubeletPodResources=false"
+- labels:
+    preset-e2e-gce-windows: "true"
+  env:
+  - name: NUM_WINDOWS_NODES
+    value: "3"
+  - name: NUM_NODES
+    value: "2"
 - labels:
     preset-e2e-gce-windows-containerd: "true"
   env:
   - name: KUBE_CONTAINER_RUNTIME
     value: "containerd"
   - name: PREPULL_TIMEOUT
     value: "30m"
+- labels:
+    preset-load-gce-windows: "true"
+  env:
+  - name: MASTER_SIZE
+    value: "n1-standard-4"
+  - name: NUM_WINDOWS_NODES
+    value: "1"
+  - name: NUM_NODES
+    value: "1"
+  - name: NODE_SIZE
+    value: "n1-standard-8"
+  - name: NODE_DISK_SIZE
+    value: "100GB"
+  # Ensure good enough architecture for master machines.
+  - name: MASTER_MIN_CPU_ARCHITECTURE
+    value: "Intel Broadwell"
+  # Bump max pods per node in Kubelet, because there are more than 10
```

Comment:
Can you remove this comment? It may have made sense in the past, but not any more.

Reply:
Done. And as load test script is merged to k8s-sigs, so applied the windows-testing repo in extra_refs